src/ontology/edits/oeo-physical.omn

```diff
@@ -3545,17 +3548,22 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/411",
 Class: OEO_00010019
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Synthetic liquid fuel is synthetic fuel produced in a power-to-liquid system from water and carbon dioxide using electric energy applying.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "PtL fuel is a liquid synthetic fuel produced in a power-to-liquid system from water and carbon dioxide using electric energy applying.",
```

Comment:
My suggestion: _PtL fuel is a liquid synthetic fuel produced in a power-to-liquid system from water and carbon dioxide using electric**al** energy ~~applying~~._
I do not understand the _applying_ in this context.
We do not have _electric energy_ implemented, only _electrical energy_ and as alternative term _electricity_

Reply:
You're right. I fix this.